### ISSUE TITLE
OAuth Error: invalid_scope

### ISSUE THREAD

**Issue by Lisennk** [SocialiteProviders/Slack#8]
`OAuth Error: invalid_scope: Cannot request both identity scopes and other scopes at the same time`

Call from controller like this:
`return Socialite::with('slack')->scopes(['incoming-webhook', 'commands', 'bot'])->redirect();`

**Comment by Taras-S** [comment]
URL when error:
```
https://slack.com/oauth/reflow?client_id=43291306993.56802672307&redirect_uri=https%3A%2F%2Fcryptocompass.dev%2Fauth%2Fslack%2Fcallback&scope=identity.basic%2Cidentity.email%2Cidentity.team%2Cidentity.avatar%2Cincoming-webhook%2Ccommands%2Cbot&response_type=code&state=obZG7NpQ33JVQA0Fb0WNpPTOAJBdPBHWiSNePFwv
```

**Comment by DraperStudio** [comment]
https://api.slack.com/docs/oauth-scopes

Section Working with Scopes

Please note that certain scopes cannot be asked for in combination with each other. For instance, you cannot request both the bot scope and the client scope.

**Comment by Taras-S** [comment]
So if i need a bot scope, how can i do that? I do not request client scope as you see, it package requests it automatically

**Comment by DraperStudio** [comment]
I can't find any details about which scope combinations are not allowed so you might need to shoot an email to the Slack support.

**Comment by DraperStudio** [comment]
> Hi there,
> 
> Thanks for writing in in. The Sign-in With Slack flow is only compatible with the identity methods. Therefore, identity methods cannot be mixed with other ones. Please make sure you've gone through our SIWS documentation:
> https://api.slack.com/docs/sign-in-with-slack
> 
> For anything else, you'll want to use our ordinary oAuth flow.
> 
> Kindly,
> 
> Claudio

**Comment by stulogy** [comment]
I am also having this issue. I don't understand how we are supposed to use the 'ordinary oAuth flow' when your code forces us to use SIWS. Can you shed some more light on how we are supposed to use the 'bot' or 'commands' scopes with your package?

**Comment by Lisennk** [comment]
@stulogy, i think its impossible. Bot i want to publish Laravel package that allow it soon, may be today :)

**Comment by stulogy** [comment]
I'm rolling back to 2.0.0 which which will hopefully solve the issues. It worked before I updated.

**Comment by crash13override** [comment]
I also had the same problem and fixed it by rolling back to 2.0.0
Thanks @stulogy for the help!
